### PR TITLE
Background Sync API

### PR DESCRIPTION
**Pull request by awebdeveloper**
- https://www.chromestatus.com/features/6170807885627392
- https://platform-status.mozilla.org/#background-sync
- https://developer.microsoft.com/en-us/microsoft-edge/platform/status/backgroundsyncapi/

### CODE REVIEW

**Comment by awebdeveloper** [comment]
any help on how to fix the total usage error and get the usage data would be welcome

File: features-json/background-sync.json [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    {
+      "url":"https://wicg.github.io/BackgroundSync/spec/",
+      "title":"Working draft or equivalent"
+    }
+  ],
+  "bugs":[
+    
+  ],
+  "categories":[
+    "JS",
```
Comment (Fyrd) [inline features-json/background-sync.json]: Invalid trailing comma here. You can use http://jsonlint.com/ to validate.

**Comment by Fyrd** [comment]
@awebdeveloper Don't worry about the usage errors or the usage data, that gets generated automatically. The only issue here is that trailing comma.

File: features-json/background-sync.json (hunk 1 above)
Comment (awebdeveloper) [reply features-json/background-sync.json]: done... Time to update the error messages ;)

**Comment by awebdeveloper** [comment]
any more info i can provide to help with data review

**Pull request merged**